op-e2e/op_geth.go

```diff
@@ -37,9 +37,17 @@ var (
 
 // OpGeth is an actor that functions as a l2 op-geth node
 // It provides useful functions for advancing and querying the chain
+
+type Node interface {
+	Close() error
+}
+
 type OpGeth struct {
-	node          *gn.Node
+	node          Node
+	opNode        *gn.Node
+	erigonNode    *ExternalEthClient
```

Comment:
As suggested in the constructor, rather than storing this explicitly, we can simply store an `EthInstance` for both `opNode` and `erigonNode`.  Also, I've been trying to scrub `erigon` anywhere I see it.  These are `external`, as there's an external geth, one for erigon in our tree, and hopefully others (nethermind, besu, etc.) at some point.

Reply:
Yes, you are right. We don't need `opNode` and `erigonNode`. Removed.